src/routers/conversations.py

```diff
@@ -326,7 +328,7 @@ async def authorize_user(
     conversation_id: str,
     k8s_auth_headers: K8sAuthHeaders,
     conversation_service: IService,
-) -> None:
+) -> str:
     """Authorize the user to access the conversation."""
```

Comment:
with the new return, I think this docstring needs an update.

Reply:
Also, this might be viewed as a violation of the single responsibility principle. What will happen if this will cause an exception and not return a str?